doc/report.md

```diff
@@ -143,133 +215,330 @@ Di seguito sono riportati i requisiti individuati durante lo studio del dominio
+
+### Sprint
+
+Sono stati realizzati un totale di 5 Sprint, alla conclusione di ognuno il team si è riunito per rilasciare un prototipo
+funzionante, ad eccezione del primo Sprint e per discutere del lavoro svolto, congiuntamente si è anche provveduto ad eseguire lo Sprint Planning dello Sprint successivo.
+
+#### Sprint 1
+
+Durante il primo sprint, il team si è concentrato sulla fase di analisi del dominio, parte fondamentale per il futuro
+sviluppo del progetto. Ogni membro del gruppo, in autonomia, ha prodotto un diagramma UML ad alto livello del model, tali diagrammi sono poi stati discussi e uniti per generare il diagramma su cui successivamente si è basata l'intera implementazione del Model. Si è inoltre iniziato a compilare il Product Backlog, a definire una prima suddivisione dei task e a preparare il repository del progetto, configurando SBT e la CI
+
+
+#### Sprint 2
+
+Durante il secondo Sprint il team si è riunito per la riunione iniziale nella quale si è discussa sia la fine del primo
+Sprint che lo Sprint Planning del secondo. Di seguito sono riportate le attività svolte durante il secondo meeting:
+
+* Chiusura di tutti i task Sprint 1
+* Definizione regole del model di base dei conigli e genetica
+* Definizione prima versione model (UML)
+* Creazione della struttura MVC con loop dell'avanzamento delle generazioni
+* Revisione difficoltà dei task
+* Product Backlog per Sprint 2
+* Sprint Planning 2
+
+#### Sprint 3
+
+Durante il terzo Sprint il team si è come al solito riunito prima di tutto per il meeting settimanale nel quale si è
+discusso sia la fine dello Sprint precedente che la definizione dello Sprint Planning del terzo. Di seguito sono
+riportate le attività svolte durante il terzo meeting:
+
+* Chiusura di tutti i task Sprint 2
+* Merge di tutte le Pull Request relative allo Sprint 2
+* Rilascio Release v0.1.0
+* Revisione difficoltà dei task
+* Product Backlog per Sprint 3 e 4
+* Sprint Planning 3 con suddivisione dei task
+* Definizione dei pannelli necessari nella GUI, per le mutazioni, per i grafici e l'albero genealogico
+* Definizione e introduzione dell'aspetto delle mutazioni
+* Definizione iniziale del report
+
+#### Sprint 4
+
+Durante il quarto Sprint il team si riunito prima di tutto per il meeting settimanale nel quale si è discusso sia la
+fine dello Sprint precedente che la definizione dello Sprint Planning del quarto. Di seguito sono riportate le attività
+svolte durante il quarto meeting:
+
+* Revisione difficoltà dei task
+* Chiusura di tutti i task Sprint 3
+* Merge di tutte le Pull Request relative allo Sprint 3
+* Rilascio Release v.0.2.0
+* Aggiornamento del Product Backlog per Sprint 4
+* Sprint Planning 4
+* Definizione della realizzazione dei fattori disturbanti
+* Definizione del restart e cambiamento di velocità della simulazione
+* Definizione della realizzazione della grafica dell'orologio
```

Comment:
Come per i due Sprint precedenti anche per il quarto, il team si è posto l'obiettivo di realizzare un prodotto concreto e in questo caso, il prototipo è molto vicino alla versione finale del sistema. Nello specifico, si sono introdotti i fattori disturbanti, sia lato Model che View e si è aggiunta la possibilità di modificare la velocità del loop e di eseguire il restart al termine di ogni simulazione. Si è inoltre voluto inserire un nuovo componente nell'interfaccia grafica: un orologio, questo permette all'utente di comprendere in che momento della generazione si trova durante la simulazione.